public/app/features/transformers/editors/FormatTimeTransformerEditor.tsx

```diff
@@ -32,9 +32,12 @@ export function FormatTimeTransfomerEditor({
   }
 
   // Format timezone options
-  const tzs = getTimeZones();
+  const tzs = getTimeZones(false);
```

Comment:
should it not be true here aswell?

Reply:
Oh man haha, that was silly of me. Thanks! 🙏

Reply:
no worries, happens to the best of us!